### DISCUSSION TITLE
Theia build fails because of latest inversify update(v 6.1.1)

### DISCUSSION THREAD

**Discussion by anilkumariiit** [eclipse-theia/theia#14430] [General] [answered]
I get below error while trying to build my app which uses Theia 1.53.0. (uses Inversify ^6.0.1). Please let me know if anyone knows how to solve this error.

WARNING in ../../node_modules/inversify/es/planning/reflection_utils.js 10:22-32
export 'getTargets' (imported as 'getTargets') was not found in '@inversifyjs/core' (module has no exports)
 @ ../../node_modules/inversify/es/planning/planner.js 11:0-100 114:31-46 116:47-74 118:69-84
 @ ../../node_modules/inversify/es/container/container.js 62:0-85 122:19-39 124:32-52 317:28-45 637:26-30
 @ ../../node_modules/inversify/es/inversify.js 4:0-50 4:0-50
 @ ./src-gen/backend/server.js 12:22-42
 @ ./src-gen/backend/main.js 77:21-40
ERROR in ../../node_modules/@inversifyjs/common/lib/esm/index.js 1:0-80
Module not found: Error: Can't resolve './services/models/LazyServiceIdentifier' in '<project_path>\node_modules\@inversifyjs\common\lib\esm'
Did you mean 'LazyServiceIdentifier.js'?
BREAKING CHANGE: The request './services/models/LazyServiceIdentifier' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ../../node_modules/inversify/es/inversify.js 2:0-60 2:0-60
 @ ./src-gen/backend/server.js 12:22-42
 @ ./src-gen/backend/main.js 77:21-40
ERROR in ../../node_modules/@inversifyjs/core/lib/esm/index.js 1:0-68

**Comment by msujew** [comment, marked as answer]
Hey @anilkumariiit,

for some reason, webpack can longer correctly find the CJS entry point into inversify. That's weird, because 6.1.1 didn't change anything about the `main` or `module` entry point. Anyway, I've filed https://github.com/inversify/InversifyJS/pull/1617 with the inversify team to fix this. Hopefully, this will be resolved with 6.1.2.

**Reply by anilkumariiit** [reply]
Thanks for fixing the issue, I am going with resolutions workaround for now
